db/migrate/20222025090441_add_references.rb

```diff
@@ -0,0 +1,8 @@
+class AddReferences < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :entities, :user, foreign_key: { to_table: :users }, index: true
+    add_reference :groups, :user, foreign_key: { to_table: :users }, index: true
```

Comment:
* According to the project requirement we are instructed not to change column names visible in the diagram. for example the foreign key for entities should be `author_id ` and not `user_id`.   you can write something like this ` t.references :author, foreign_key: { to_table: 'users' }) 
![image](https://user-images.githubusercontent.com/52497246/198211758-bd98109f-d89a-4714-9b98-25fbe411da35.png)